InvenTree/plugin/test_api.py

```diff
@@ -41,6 +41,8 @@ def test_plugin_install(self):
             expected_code=201,
         ).data
 
+        print("A:", data)
```

Comment:
@SchrodingersGat I think you forgot the debug comment here?

Reply:
Ah, nice spot, thanks